core/src/main/java/jenkins/util/TimeDuration.java

```diff
@@ -42,7 +42,7 @@ public long getTimeInMillis() {
     }
 
     /**
-     * Returns the duration of this instance in <em>milliseconds</em>.
+     * Returns the duration of this instance in <em>seconds</em>.
```

Comment:
Better (IMHO), `<code />`

Reply:
This PR being a small fix for #3043 I don't want anyway to change the semantics. 
So, If I wanted to do that, I'd do on a separate PR. 

Now, on that subject itself, I don't think `<code>` should be used in that case, since that text "milliseconds" is not code (And I'm not even talking about `{@code ...}` vs `<code />` :-)). "milliseconds" or "seconds" is simply some text I want to *em*phasize a little more than the rest.

Reply:
@batmat Yes, you are right.